translations/de/Info/task_description.html

```diff
@@ -0,0 +1,62 @@
+<img class="for_info_only" title="base" src="{{MEDIA}}base.png" alt="base"
+     style="width: 300px;float: right; margin-top: 30px; padding: 0 10px 0 30px;"/>
+
+<p>
+    Erinnerst du dich an<a href="https://de.wikipedia.org/wiki/Stellenwertsystem#Basis"> die Basiszahl</a> und
+    <a href="https://de.wikipedia.org/wiki/Stellenwertsystem">Stellenwertsysteme</a> vom Mathe-Unterricht ?
+    Lass uns damit diese Aufgabe bestreiten.
+</p>
+
+<p>
+    Du bekommst eine positive Zahl als Zeichenfolge(String) mit der zugehörigen Basiszahl. Deine Funktion soll sie ins 
+    Dezimalsystem umwandeln.
+    Die Basis ist weniger als 37 und grösser als 1.
+    Die Aufgabe verwendet Ziffern und die Buchstaben A-Z für die Zeichenfolgen.
+</p>
+
+<p>
+    Pass auf für Fälle wenn es nicht möglich ist die Zahl umzuwandeln.
+    Zum Beispiel: "1A" kann nicht mir Basis 9 umgewandelt werden.
+    Für diese Fälle soll deine Funktion -1 zurückgeben.
+</p>
+
+<p>
+    <strong>Eingabe: </strong> Zwei Argumente. Eine Nummer als Zeichenfolge(String) und eine Basis als Ganzzahl(Integer).
+</p>
+
+<p>
+    <strong>Ausgabe: </strong> Die umgewandelte Zahl als Ganzzahl(Integer).
+</p>
+
+
+<div class="for_info_only">
+    <p>
+        <strong>Beispiel:</strong>
+    </p>
+    <pre class="brush: python">
+checkio("AF", 16) == 175
+checkio("101", 2) == 5
+checkio("101", 5) == 26
+checkio("Z", 36) == 35
+checkio("AB", 10) == -1
+    </pre>
+</div>
+
+<p class="for_info_only">
+    <strong>Für was soll ich das tun: </strong>
+    Hiermit lernst du wie du mit verschiedenen Zahlensystemen arbeiten kannst und wie du mit Ausnahmen umgehst.
```

Comment:
complete line should read as: "Hiermit lernst Du, wie Du mit verschiedenen Zahlensystemen arbeiten kannst und wie Du mit Ausnahmen umgehst." (correction by Duden Rechtschreibprüfung)